nessie/multipart_test.go

```diff
@@ -47,9 +54,10 @@ func TestMultipartUpload(t *testing.T) {
 			WithRef(masterBranch).
 			WithPath(file), nil, &b)
 	require.NoError(t, err, "failed to get object")
+	require.True(t, bytes.Equal(b.Bytes(), partsConcat), "uploaded object did not match")
```

Comment:
Can you instead use testify's Equal method, it should give better error reporting (and also be shorter)?
```suggestion
	require.Equal(t, b.Bytes(), partsConcat, "uploaded object did not match")
```

Reply:
Initially, I thought `Equals` will not work on slices (reference types). But in docs I found it does.